macros/GroupData.json

```diff
@@ -1834,10 +1857,15 @@
                             "RTCSessionDescriptionCallback",
                             "RTCStatsReport",
                             "RTCCertificate",
+                            "RTCIdentityProvider",
+                            "RTCIdentityAssertion",
```

Comment:
This is a dictionary, is it an important one? Not sure how useful it is to also list all the dictionaries in the navigation